cts/scheduler/xml/bug-1572-1.xml

```diff
@@ -54,9 +54,6 @@
         </meta_attributes>
       </clone>
       <group id="grp_pgsql_mirror">
-        <meta_attributes id="meta_attributes.id22200">
-          <nvpair id="nvpair.id22206" name="restart-type" value="restart"/>
-        </meta_attributes>
```

Comment:
I think we do want to replace/transform restart=restart-type to kind=Mandatory when possible. That's the closest behavior. If anyone still has restart-type left over from circa 2008 (highly unlikely), it's probably what they really wanted.

Reply:
Yeah, I stand by the transform not being worth the effort... We'd have to contend with (or ignore) at least the following that would affect behavior:
* rules
* resource sets
* different `restart-type` for `first` and `then`
* explicit `kind`

Reply:
I'll update regression tests, except that if we have explicit `kind="Optional"`, I'll leave that in place and simply drop `restart-type="restart"`.

Reply:
*All* of them have explicit `kind="Optional"`.

Reply:
Well... dropping `kind="Optional"` basically preserves the test outputs in these cases, so I will do that. Apparently the `restart-type` took precedence. There seems to be an asymmetry to it though.